### ISSUE TITLE
command remote_start_drive exception {"response":null,"error":"user not found","error_description":""} 

### ISSUE THREAD

**Issue by joinwangzi** [mseminatore/TeslaJS#320]
request: https://owner-api.teslamotors.com/api/1/vehicles/xxxxxxxx/command/remote_start_drive
response : {"response":null,"error":"user not found","error_description":""}

I am in China, It was normal for me to use the remote start command before March 26, 2022, and then I used the remote start command again on March 29, 2022 and found that it could not be used normally. It will return {"reason": "", "result": true}.

However, in recent days, it has been found that this API interface cannot be used normally. It returns {"response": null, "error": "user not found", "error_description": ""}.
I confirm that the ID and password are correct.
Other commands can be used normally, such as wake_up、honk_horn、flash_lights.

I tried to refresh the token and change the login password, but it still didn't work.
Is anyone in this situation? Please help me! Thanks

**Comment by mseminatore** [comment]
We have been having some issues with the library since Tesla changed the authentication process to require a captcha. I have so far been unable to fix the library. It sounds like you able to use other API calls successfully so you are able to generate your own tokens?

**Comment by joinwangzi** [comment]
Yes, for example, other double flashing, honking, opening the door, closing the door, and waking up can be done. The 45-day token is no longer used. Now, the 8-hour token is refreshed with refresh_token. Have you tried remote startup and it still works normally?



---- 回复的原邮件 ----
| 发件人 | Mark ***@***.***> |
| 日期 | 2022年04月28日 05:28 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [mseminatore/TeslaJS] command remote_start_drive exception {"response":null,"error":"user not found","error_description":""} (Issue #320) |

We have been having some issues with the library since Tesla changed the authentication process to require a captcha. I have so far been unable to fix the library. It sounds like you able to use other API calls successfully so you are able to generate your own tokens?
